### ISSUE TITLE
Allow multiple sbt instances on a project.

### ISSUE THREAD

**Issue by sellout** [ensime/emacs-sbt-mode#97]
Something like how eshell works, where a (negative?) prefix arg opens a new buffer rather than jumping to an existing one.

**Comment by fommil** [comment]
Is this to allow you to do multiple things at once?

I've never had reason for a second one.

**Comment by sellout** [comment]
Yeah, exactly. I don’t need it _often_, but sometimes it’s very useful. For example, yesterday I was running `partest` on the scala codebase, and it takes a long time. But when a test failed, I wanted to be able to open a second sbt and edit/rerun just that one test without interrupting the overall test suite.

**Comment by fommil** [comment]
try this alternative: rename the long running sbt buffer to something else. Then start a new one. Then when the long running task is finished, throw it away.

**Comment by sellout** [comment]
So, this was mitigated by @fommil’s suggestion above, but the problem I continue to have is that when I try to run a command like `sbt-clear` or something on the renamed buffer, it actually performs the command on the _other_ buffer.

So, the way that the “current” sbt buffer is found should be modified to first see if the current buffer is an sbt buffer, and if so, not look for the “primary” sbt buffer for the project.
